### DISCUSSION TITLE
Allow cookies in session when logging in with redirection

### DISCUSSION THREAD

**Discussion by reggZez** [cypress-io/cypress#23279] [Questions and Help]
Hey,
Don't know if the title is correct but let me describe my problem. I am having trouble logging into my application using microsoft login. How it works manually, is that once you go to the app URL you are redirected to the Microsoft login and after inserting valid credentials you are redirected back to the app.
After I supply valid url with cy.visit('/') to my app, I should be redirected to the Microsoft login, but instead I get that my browser is blocking cookies.
![image](https://user-images.githubusercontent.com/111056328/184122085-bd5c5414-1f89-4394-9b81-4c7f98a647de.png)

Previous issue that I had with this was the removal of Cypress test frame upon redirection. This is resolved by using:  experimentalModifyObstructiveThirdPartyCode
experimentalSessionAndOrigin

Is there a way to login into the app with all these redirections?  Any information would be helpful.

**Comment by Krooksy** [comment]
Having the exact same issue, very interested to see if anyone has gotten this to work using cy.session.
